changes/4937.dependencies

```diff
@@ -0,0 +1,5 @@
+Updated alpha UI dependency @rjsf/core to version 5.13.6.
```

Comment:
I used alpha because that's what we called it in the setting [`ENABLE_ALPHA_UI`](https://github.com/nautobot/nautobot/blob/e34d32ef629481d601e9d8ce3195bcdd81c19e52/nautobot/core/settings.py#L917)